src/ffem.yaml

```diff
@@ -0,0 +1,26 @@
+---
+description: >
+  The Foundation for Environmental Monitoring is a nonprofit company developing
+  open source products including hardware designed to help address issues
+  related to water and soil
+license: GPL 3.0
+name: ffem
+programmingLanguages:
+  - java
+  - python
+  - javascript
+  - html
+  - kotlin
+rating: 5
+repoUrl: "https://github.com/foundation-for-environmental-monitoring"
+sdgs: [6, 13, 14, 15]
```

Comment:
I looked through their projects (on their website) and I'm not sure how it relates to 13: Climate action.
Could you please quickly point out what project relates to 13 (apart from the name)?

Reply:
I didn't think too much about it and went with the naive assumption that any action related to water and soil has a direct or indirect impact on Climate action (soil being the largest store of carbon, which overexploitation releases lot of carbon dioxide in the air.